File: Sem1/Introducere în programare/Tutorials/W1Ex11.c
#include<stdio.h>
int main(void) 
{
int x, y, sum=0;
printf("Cate numere vrem?");
scanf("%d", &x);

for(int i=0; i<x; i++) {
printf("Introduceti un numar: ");
scanf("%d", &i);
sum+=i;
}

printf("Suma tuturor numerelor pozitive citite de la tastatura este %d.\n", sum);

return 0;
}

File: Sem1/Introducere în programare/Tutorials/W1Ex11.php
<?php
echo "Cate numere?";
$x = readline();
$suma=0;

for ($i = 0; $i < $x; $i++)
{
echo"y=";
$y=readline();
if($y > 0)
{
$suma+=$y;
}
}
echo "Suma = $suma";